plugins/check_http.c

```diff
@@ -1257,32 +1242,63 @@ check_http (void)
   }
 
   /* Page and Header content checks go here */
-  if (strlen (header_expect)) {
-    if (!strstr (header, header_expect)) {
-      strncpy(&output_header_search[0],header_expect,sizeof(output_header_search));
-      if(output_header_search[sizeof(output_header_search)-1]!='\0') {
-        bcopy("...",&output_header_search[sizeof(output_header_search)-4],4);
+  if (strlen(header_expect) > 0) {
+    if (strstr(header, header_expect) == NULL) {
+      // We did not find the header, the rest is for building the output and setting the state
+      char output_header_search[30] = "";
+      strncpy(&output_header_search[0], header_expect,
+              sizeof(output_header_search));
+      if (output_header_search[sizeof(output_header_search) - 1] != '\0') {
+        bcopy("...", &output_header_search[sizeof(output_header_search) - 4],
+              4);
```

Comment:
tbh, i don't think this is more readable than before.

Reply:
i agree with the two if statements, but how is moving three characters to the next line making things more readable?